py-scripts/test_l3.py

```diff
@@ -6569,7 +6725,7 @@ def main():
     test_l3_parser.add_argument(
         '--dowebgui',
         help='--dowebgui True  if running through webgui',
-        action='store_true')
```

Comment:
No need to fix just FYI. This change doesn't do anything. `action='store_true'` implies that the argument is a boolean.